tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/LockFreeExponentiallyDecayingReservoir.java

```diff
@@ -227,6 +247,10 @@ private Builder() {}
          * replaced (based on weight, with some amount of random jitter).
          */
         public Builder size(int value) {
+            if (value <= 0) {
+                throw new SafeIllegalArgumentException(
```

Comment:
modification: Using SafeIllegalArgumentException with a SafeArg

Reply:
I manually retained safe-logging preconditions on 270 and 278 in place of Objects.requireNonNull